products/pdt/cd/overlays/staging/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base
+  - certificate.yaml
+
+patches:
+  - target:
+      kind: Ingress
```

Comment:
I don't think this target hits anything?

Reply:
yes it probably doesn't can i then remove everything related to ingress and certificates?